src/lib/server/runtime-config.ts

```diff
@@ -185,6 +185,7 @@ const baseConfig = {
 	displayCompanyInfo: false,
 	displayMainShopInfo: false,
 	disableNostrBotIntro: false,
+	hideFromSearchEngines: true,
```

Comment:
```suggestion
	hideFromSearchEngines: false,
```

we don't want to hide by default